IFC4x3/Properties/p/Pressiometer type_1y3JjznIn4hQPFaA5mCFy3/Documentation.md

```diff
@@ -0,0 +1 @@
+Test type (equipment).
```

Comment:
Is this description okay? It does not say much to me in terms of explaining the property.